specifications/api/swagger.yaml

```diff
@@ -321,6 +326,30 @@ x-poolMetadataSource: &poolMetadataSource
       format: uri
       example: https://smash.cardano-mainnet.iohk.io/
 
+x-walletCredentialPubkey: &walletCredentialPubkey
+  description: An extended public key (public key + chain code) for credential
```

Comment:
Why require an extended key here? The chain code is unnecessary.

Reply:
to my best knowledge we are hashing extended public key with blake2b not just public key -> https://github.com/input-output-hk/cardano-addresses/blob/master/core/lib/Cardano/Address/Style/Shelley.hs#L639

Reply:
Not we're not :) 

https://github.com/input-output-hk/cardano-addresses/blob/1939121afcd125b6c6300a1b2a8b014007a0356c/core/lib/Cardano/Address/Style/Shelley.hs#L869-L872

`xpubPublicKey` is the public key, without chain code.

Reply:
indeed! Thanks! I made this mistake mainly because of this : 
```
$ cat signingKey1.prv | cardano-address key public --bech32
xpub13pfp60uf4ru7cmj5d57xjdzj6lagn328ah67h35urqrl92jn094z0s07fcy2pvfw3xfhna6fa24na8nk6lraymjcrl5wwj6chsjqtjqlfw44z
```
So we will need a way to get public key only (not chain code included) in cardano-addresses, right?

Reply:
I think that'd be an interesting feature indeed. Be able to get just the public part, non extended.

Reply:
Alternatively, we can have the API accepts _both_, but the chain code should not a _required_

Reply:
corrected in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/2f9180366bdb4319b19c3ad90bb06e5f52fc350e